### ISSUE TITLE
Docker Synology DSM 7.X

### ISSUE THREAD

**Issue by 0n1cOn3** [kopiro/homebridge-playstation#33]
**Describe The Bug:**

Theres no root access for the homebridge-ui
**To Reproduce:**

Install Docker on Synology (or choose xpenology + vbox)
Install Homebridge SPK file
Install your Plugin
Try to configure any PS

**Expected behavior:**

To work as described as written in the guide :D 
**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```
ERROR: sudo / root access is not allowed by the homebridge service account. See https://kb.synology.com/en-id/DSM/tutorial/How_to_login_to_DSM_with_root_permission_via_SSH_Telnet

And, yeah theres no account available as shown in the Syno's Description.

**Plugin Config:**

```json
            "pollInterval": 5000,
            "platform": "playstation",
            "_bridge": {
                "username": "xx:xx:xx:xx:xx",
                "port": 32374
```

**Screenshots:**

**Environment:**

* **Plugin Version**: Latest
* **Homebridge Version**:  Latest
* **Node.js Version**:  Latest
* **NPM Version**:  Latest
* **Operating System**:  DSM 7.0.1-42218 Update 3

**Comment by kopiro** [comment]
Hello; i'm not really familiar with your setup; do you have a `root` user available?

**Comment by 0n1cOn3** [comment]
> Hello; i'm not really familiar with your setup; do you have a `root` user available?

Hi

Yes, it exists 😄

**Comment by nobugsincode** [comment]
@0n1cOn3 I had the same problem and I was able to fix it. So what I did:
1. Installed ps4-waker npm package to my MacBook
2. Run ps4-waker and followed the instructions so the utility created a file ~/.ps4-waker.credentials.json
3. Installed playactor npm package to my MacBook
4. Run playactor login so it was able to pick existing creds and created it's own version in ~/.config/playactor/credentials.json
5. Installed playactor to Synology via Homebridge terminal
6. Copied credentials file from 4 to ~/.config/playactor/credentials.json on my NAS
7. Run playactor login on NAS (still in same HB terminal) and followed the instructions
8. Profit :)

**Comment by 0n1cOn3** [comment]
> @0n1cOn3 I had the same problem and I was able to fix it. So what I did:
> 
> 1. Installed ps4-waker npm package to my MacBook
> 
> 2. Run ps4-waker and followed the instructions so the utility created a file ~/.ps4-waker.credentials.json
> 
> 3. Installed playactor npm package to my MacBook
> 
> 4. Run playactor login so it was able to pick existing creds and created it's own version in ~/.config/playactor/credentials.json
> 
> 5. Installed playactor to Synology via Homebridge terminal
> 
> 6. Copied credentials file from 4 to ~/.config/playactor/credentials.json on my NAS
> 
> 7. Run playactor login on NAS (still in same HB terminal) and followed the instructions
> 
> 8. Profit :)

Thanks for the Workaround! Will try that out and let you guys know if it has worked. Maybe this steps could be added to the Guide later ¯\_(ツ)_/¯

**Comment by yahms** [comment]
> @0n1cOn3 I had the same problem and I was able to fix it. So what I did:
> 
> 1. Installed ps4-waker npm package to my MacBook
> 2. Run ps4-waker and followed the instructions so the utility created a file ~/.ps4-waker.credentials.json
> 3. Installed playactor npm package to my MacBook
> 4. Run playactor login so it was able to pick existing creds and created it's own version in ~/.config/playactor/credentials.json
> 5. Installed playactor to Synology via Homebridge terminal
> 6. Copied credentials file from 4 to ~/.config/playactor/credentials.json on my NAS
> 7. Run playactor login on NAS (still in same HB terminal) and followed the instructions
> 8. Profit :)

yeah thanks this helped me too.. i did the following. I run hb in docker on synology, using the inbuilt docker (yes yes, i know, im too lazy for portainer) tho i skip the SPK and just use a scheduled docker run command to refresh the container

1. Log in to Homebridge UI, Open Terminal, these commands (the user you run as may depend on ur config?):
2. `npm install ps4-waker -g`
3. `ps4-waker`
4. Followed the process (it appeared in the PS4 App on phone)
5. make sure this file exists `cat ~/.ps4-wake.credentials.json`
6. `mkdir ~/.config`
7. `mkdir ~/.config/playactor`
8. `cp .ps4-wake.credentials.json ./.config/playactor/credentials.json`
9. `homebridge-playstation-login`
10. Follow that process, though it shows in the PS as ps4 waker, i guess because thats what authenticated
11. (i also saw some error that it didnt work or something, but I just ignored that and carried on lol)
12. Restarted homebridge
13. Added accessory
14. ???
15. Profit
16. optional: `npm uninstall ps4-waker` - on mine it still works without that module. not needed anymore, and coz its not part of HB and wont be updated, i removed it. easy enough to install later
